INSTALL.md

```diff
@@ -4,34 +4,32 @@ If you note other requirements, feel free to add it here.
 
 ## GeoKrety Website requirements
 
-* apache / apache2 + mod_rewrite
-* php
-* imagemagick
-* smarty 2 + smarty-gettext plugin
+Geokrety website rely on docker containers with different images:
+* Geokrety PHP 7.3 base image : cf. [geokrety-website-docker-base](https://github.com/geokrety/geokrety-website-docker-base)
+* Website reuse this image withing [Dockerfile](./DockerFile)
 
+As Geokrety contributors, you must have some requirements. Especially:
+* Docker, Makefile,
+* Php (curl + composer, phpcs) : cf. OS dedicated sections below.
 
-## Run Geokrety locally (gkShell)
-
-To run a geokrety instance on your Windows/Linux workstation using Docker :
-
- cf. [gkShell](gkShell/README.md)
 
+## Run Geokrety locally
 
-## Run Geokrety locally (docker-compose)
-
-(Tested on Windows 10)
+To run a geokrety instance on your Windows/Linux workstation using Docker :
 
 Pre-requisites:
-- Docker installed
-- Docker-compose installed
+- Docker, Docker-compose installed
+- *geokrety-website-docker-base* image available 
 
-One-line run command:
+```
+ make build
+ make start
+```
 
-     docker-compose -f docker-compose.local.yml up
+Then local instance should be up and running and available at http://localhost:8000/
 
-Now local instance should be up and running and available at http://localhost:8000/
-
-Empty database with proper schema was initialized and exposed on local port `13306`.
+An empty database with proper schema was initialized and exposed on local port `13306`. 
+You could initialize some data using Makefile `seed` target.
 
 Errors are logged in `gk-errory` table in the database
```

Comment:
This is not true anymore in `feature/new-theme`

Reply:
ok I ll remove that